smoke-tests/apps/AzureSdk/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/AzureSdkControllerSpansEnabledTest.java

```diff
@@ -55,7 +55,8 @@ void test() throws Exception {
     assertThat(telemetry.rd.getResponseCode()).isEqualTo("200");
     assertThat(telemetry.rd.getSuccess()).isTrue();
     assertThat(telemetry.rd.getSource()).isNull();
-    assertThat(telemetry.rd.getProperties()).isEmpty();
+    assertThat(telemetry.rd.getProperties().get("_MS.ProcessedByMetricExtractors"))
+        .isEqualTo("True");
```

Comment:
can you add this everywhere? helps to catch if we accidentally add new properties
```suggestion
    assertThat(telemetry.rd.getProperties()).hasSize(1);
    assertThat(telemetry.rd.getProperties().get("_MS.ProcessedByMetricExtractors"))
        .isEqualTo("True");
```